### PR TITLE
add expendable columns + expendable tables + some UI teaks

### PR DESCRIPTION
**Pull request by evans-g-crsj**
image

### CODE REVIEW

File: src/components/ExpandableCell.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
```
Comment (evans-g-crsj) [inline src/components/ExpandableCell.tsx:1]: similar components. They should be merged after webinar.

**Review by adipaul1981 — APPROVED** [review]

File: src/components/ExpandableCell.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+const DEFAULT_NUM_COLLAPSED = 5;
+
+const ExpandableCell = ({ numWhenCollapsed, dataSource, renderItem, ...listProps }: OwnProps) => {
+  const [showAll, setShowAll] = useState(false);
+
+  const nLimit = numWhenCollapsed || DEFAULT_NUM_COLLAPSED;
+
+  const sliceNum = showAll ? undefined : nLimit;
+
+  const showButton = (dataSource?.length || 0) > nLimit;
```
Comment (francisl) [inline src/components/ExpandableCell.tsx:19]: why so many `\n`
plus you don't do that patten in your other file

File: src/components/ExpandableCell.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { Button, List, Space } from 'antd';
+import { ListProps } from 'antd/lib/list';
+import { CaretDownOutlined, CaretUpOutlined } from '@ant-design/icons';
+
+type OwnProps = ListProps<any> & {
+  numWhenCollapsed?: number;
```
Comment (francisl) [inline src/components/ExpandableCell.tsx:7]: what does that do?

File: src/pages/variantEntity/TabSummary.tsx [hunk 4]
```diff
@@ -188,8 +229,18 @@ const TabSummary = ({ variantId }: OwnProps) => {
...
-              <Card key={index} title={[symbol, omim, biotype].join(' ')}>
-                <Table dataSource={makeRows(orderedConsequences)} columns={columns} />
+              <Card key={index}>
+                <ExpandableTable
+                  numWhenCollapsed={1}
+                  buttonText={(showAll, hiddenNum) =>
+                    showAll ? 'Less' : `Show Transcripts (${hiddenNum})`
+                  }
+                  key={index}
+                  title={() => [symbol, omim, biotype].join(' ')}
```
Comment (francisl) [inline src/pages/variantEntity/TabSummary.tsx:239]: title should go in card, no?

File: src/pages/variantEntity/TabSummary.tsx [hunk 5]
```diff
@@ -102,22 +113,52 @@ const columns = [
   {
     title: 'Strand',
     dataIndex: 'strand',
+    render(strand: any) {
```
Comment (francisl) [inline src/pages/variantEntity/TabSummary.tsx:116]: any?

**Review by francisl — CHANGES_REQUESTED** [review]

**Review by francisl — APPROVED** [review]

**Pull request merged**